data/insights/general/domain.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.1"
+
+intents:
+  - intent_setup_insights
+
+actions:
+  - validate_form_setup_insights
+
+slots:
+  insights_product_kind:
+    type: categorical
+    values:
+      - rhel
+      - openshift
+      - ansible
+    mappings:
+      - type: custom
+
+forms:
+  form_setup_insights:
+    required_slots:
+      - insights_product_kind
+
+responses:
+  utter_ask_insights_product_kind:
+    - text: "Are you looking for help with Insights for Ansible, Openshift, or RHEL?"
+      buttons:
+        - title: Ansible
+          payload: ansible
+        - title: Openshift
+          payload: openshift
+        - title: RHEL
+          payload: rhel
+  utter_ansible_activation_guide:
```

Comment:
A small thing, but all these intents should have insights as a prefix (since the openshift_docs one would be especially easy to conflict)